timelock-impl/src/main/java/com/palantir/atlasdb/timelock/batch/MultiClientConjureTimelockResource.java

```diff
@@ -74,6 +77,29 @@ public static MultiClientConjureTimelockService jersey(
                 MoreExecutors.directExecutor()));
     }
 
+    @Override
+    public ListenableFuture<Map<Namespace, ConjureStartTransactionsResponse>> startTransactions(
+            AuthHeader authHeader, Map<Namespace, ConjureStartTransactionsRequest> requests) {
+        List<ListenableFuture<Map.Entry<Namespace, ConjureStartTransactionsResponse>>> futures = KeyedStream.stream(
+                        requests)
+                .map(this::getStartTransactionsResponseListenableFutures)
+                .values()
+                .collect(Collectors.toList());
+        return handleExceptions(() ->
+                Futures.transform(Futures.allAsList(futures), ImmutableMap::copyOf, MoreExecutors.directExecutor()));
+    }
+
+    private ListenableFuture<Map.Entry<Namespace, ConjureStartTransactionsResponse>>
+            getStartTransactionsResponseListenableFutures(
```

Comment:
Method looks right as far as I can tell, but I think the name is a bit off. Maybe just `startTransactionsForSingleNamespace` or something?